staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/validation/validation.go

```diff
@@ -51,6 +51,7 @@ func ValidateLabelSelectorRequirement(sr metav1.LabelSelectorRequirement, fldPat
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("operator"), sr.Operator, "not a valid selector operator"))
 	}
 	allErrs = append(allErrs, ValidateLabelName(sr.Key, fldPath.Child("key"))...)
+	allErrs = append(allErrs, ValidateLabelValues(sr.Values, fldPath.Child("values"))...)
```

Comment:
Any time we want to tighten validation of an existing field, we need to answer the following questions:
* What objects use this validation?
* What will tightening validation do if existing objects have invalid values?
* Will it prevent status updates by controllers?
* Do the things that automatically create/update this object tolerate validation rejections well? (backoff, surface error to user)